sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosContainerProperties.java

```diff
@@ -211,33 +211,76 @@ public Integer getDefaultTimeToLiveInSeconds() {
     /**
      * Sets the collection's default time-to-live value.
      * <p>
-     * The default time-to-live value on a collection is an optional property. If set, the documents within the 
+     * The default time-to-live value on a collection is an optional property. If set, the documents within the
      * collection
-     * expires after the specified number of seconds since their last write time. The value of this property should 
+     * expires after the specified number of seconds since their last write time. The value of this property should
      * be one of the following:
      * <p>
      * null - indicates evaluation of time-to-live is disabled and documents within the collection will never expire,
      * regardless whether
      * individual documents have their time-to-live set.
      * <p>
-     * nonzero positive integer - indicates the default time-to-live value for all documents within the collection. 
+     * nonzero positive integer - indicates the default time-to-live value for all documents within the collection.
      * This value can be overridden
      * by individual documents' time-to-live value.
      * <p>
-     * -1 - indicates by default all documents within the collection never expire. This value can be overridden by 
+     * -1 - indicates by default all documents within the collection never expire. This value can be overridden by
      * individual documents'
      * time-to-live value.
      *
      * @param timeToLive the default time-to-live value in seconds.
      */
-    public void setDefaultTimeToLiveInSeconds(Integer timeToLive) {
+    public CosmosContainerProperties setDefaultTimeToLiveInSeconds(Integer timeToLive) {
```

Comment:
Nice catch, are there any such left overs?

Reply:
seems there are a few other places as well.

@kushagraThapar  FYI
```bash
find . | egrep "Properties|Setting|Config|Option" | grep -v implementation| xargs grep void
./java/com/azure/cosmos/models/CosmosStoredProcedureProperties.java:    public void setBody(String body) {
./java/com/azure/cosmos/models/FeedOptions.java:     * was done but not yet utilized. This avoids redoing the work again in
./java/com/azure/cosmos/models/FeedOptions.java:    public void setEmptyPagesAllowed(boolean emptyPagesAllowed) {
```

Reply:
ok I fixed all of them in this PR.